MyArrayList/Program.cs

```diff
@@ -215,6 +234,7 @@ public void Delete(int index)
         }
         Array.Resize(ref _list, _list.Length - 1);
     }
+    
     //Adds element to the particular index
     public void Add(int num, int index)
```

Comment:
The better name for this method is InsertToIndex - it is seft-described, so there will be no need in commenting it.